docs/README.md

```diff
@@ -1,6 +1,5 @@
 <!--
 	Are you editing this file?
-	* /README.md and docs/README.md should be identical copies (symlinks don't work)
 	* Use only *absolute links* in these files. Relative links will break.
```

Comment:
I'm not sure if this one is still relevant.

Reply:
I think this whole comment can die